pkg/network/ebpf/c/shared-libraries/probes.h

```diff
@@ -133,6 +153,8 @@ int tracepoint__syscalls__sys_exit_openat(exit_sys_ctx *args) {
 SEC("tracepoint/syscalls/sys_enter_openat2")
 int tracepoint__syscalls__sys_enter_openat2(enter_sys_openat2_ctx *args) {
     CHECK_BPF_PROGRAM_BYPASSED()
+    // Unlike the other variants, openat2(2) has the flags embedded inside the
+    // how argument; we don't bother trying to accessing it for now.
```

Comment:
Why skip it? From what I know, libc prefers openat2 so you would get more benefit from covering this syscall.

Reply:
I've skipped it for now since it's somewhat more effort than the other ones which just needed the check on the already available field, since it needs us to read one more struct. glibc doesn't support openat2 yet: https://sourceware.org/bugzilla/show_bug.cgi?id=31664